### PR TITLE
Add documentation to trust_SignTransaction method

### PR DESCRIPTION
**Pull request by leoneparise**

### CODE REVIEW

File: wallet-connect/dapp.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      }
+    ]
+  }
+};
+
+walletConnector
+  .trusSignTransaction(network, tx)
+  .then(result => {
+    // Returns transaction signed in json or encoded format
+    console.log(result);
```
Comment (vikmeup) [inline wallet-connect/dapp.md]: can you include result format? for more details?

File: wallet-connect/dapp.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  // Delete walletConnector
+});
+```
+
+### Sign Transaction
+Once you have `walletconnect client` set up, you can sign a transaction:
+
+```javascript
+const network = 118; // Atom (SLIP-44)
+// Transaction structure based on Trust's protobuf messages.
```
Comment (vikmeup) [inline wallet-connect/dapp.md]: can you link `Trust's protobuf messages` to `https://github.com/TrustWallet/wallet-core/tree/master/src/proto` as the link below

File: wallet-connect/dapp.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# DApp Integration
+
+Trust uses a forked version of WalletConnect with aditional methods to allow **dApp** developers
+to sign transactions for any blockchain. 
+
+## Getting started
+To use Trust's WalletConnect implementation, you need to clone the following repository:
+
+```bash
+git clone git@github.com:TrustWallet/walletconnect-monorepo.git
```
Comment (vikmeup) [inline wallet-connect/dapp.md]: is it required to have forked version of trust to use `trust_sign` ?

File: wallet-connect/dapp.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
+# DApp Integration
+
+Trust uses a forked version of WalletConnect with aditional methods to allow **dApp** developers
+to sign transactions for any blockchain.
```
Comment (vikmeup) [inline wallet-connect/dapp.md]: for any blockchain. - can you link to this url: https://github.com/TrustWallet/wallet-core/blob/master/docs/coins.md ?

File: wallet-connect/dapp.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+```bash
+cd packages/core
+npm install
+npm run build
+npm link
+```
+
+and build and link `@walletconnect/browser` package:
+
+```bash
```
Comment (vikmeup) [inline wallet-connect/dapp.md]: should we have this tutorial on wallet core page for installing?

File: wallet-connect/dapp.md (hunk 3 above)
Comment (leoneparise) [reply wallet-connect/dapp.md]: Yes. It is. I didn't submit the pull request to walletconnect.

File: wallet-connect/dapp.md (hunk 2 above)
Comment (leoneparise) [reply wallet-connect/dapp.md]: I don't think we're able to add links to a code block.

File: wallet-connect/dapp.md [hunk 6]
```diff
@@ -3,6 +3,8 @@
 Trust uses a forked version of WalletConnect with aditional methods to allow **dApp** developers
 to sign transactions [for any blockchain](https://github.com/TrustWallet/wallet-core/blob/master/docs/coins.md). 
 
+> We're currently supporting **Ethereum**, **Binance Coin** and **Cosmos**. More chains will be added soon.
```
Comment (vikmeup) [inline wallet-connect/dapp.md]: can you use our standard supported blockchain block: `https://github.com/TrustWallet/blockatlas#block-atlas-by-trust-wallet`, same used for WalletCore

**Pull request merged**